pages/pipelines/group_step.md.erb

```diff
@@ -185,6 +185,8 @@ steps:
 ```
 {: codeblock-file="pipeline.yml"}
 
+Remember that a wait step placed inside a set of `commands` will be ignored because <code>wait</code> is not a command.
```

Comment:
```suggestion
```

Yeah this is the thing that that I don't want to call out *here*. If we're gonna call it out it out it should be here 

https://buildkite.com/docs/pipelines/command-step#command-step-attributes

> The shell command/s to run during this step. This can be a single line of commands, or a list of commands that must all pass. Also available as the alias commands, in which case you can use multiple commands but not other step types. 

kinda thing